src/main/Resources/traffic.csv

```diff
@@ -0,0 +1,13 @@
+Car, WantsToDrive

```

Comment:
The folder should be called `resources`, not `Resources` (capitalization matters).